lib/index.js

```diff
@@ -204,13 +207,9 @@ module.exports.render = function(options) {
   var importer = options.importer;
 
   if (importer) {
-    options.importer = function(file, prev, key) {
+    options.importer = function(file, prev, bridge) {
       function done(data) {
-        console.log(data); // ugly hack
```

Comment:
This is history attached to it. Without this, it fails with CLI usage and manually calling render with importer. If you have find a way to fix it. Kudos! But please make sure of it.